covidapi/import_data.py

```diff
@@ -6,9 +6,19 @@
 from dataflows import Flow, load, checkpoint
 from collections import defaultdict
 from tabulator.exceptions import HTTPError
+import argparse
 
 BASE_URL = 'https://raw.githubusercontent.com/CSSEGISandData/COVID-19/master/csse_covid_19_data/csse_covid_19_daily_reports/'
 
+parser = argparse.ArgumentParser(description='Import data into the database')
+parser.add_argument(
+    '--from-date',
+    metavar='DATE',
+    type=date.fromisoformat,
+    default=date(year=2020, month=2, day=29),
+    help='date to start importing from (default: 2020-02-29)'
```

Comment:
This way we can more easily tell our import script to import data from (for example) yesterday, so it will update/insert only the last couple of days, right?

Reply:
Yep that's the idea